analysis/estimate/run-estimate-analysis.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script runs an analysis using ESTIMATE
+
+set -euo pipefail
+
+# Run script from its location
+basedir=$(dirname "${BASH_SOURCE[0]}")
+cd "$basedir"
+
+# Define directories
+data_dir="data"
+script_dir="scripts"
+scpca_dir="${data_dir}/scpca"
+tpm_dir="${data_dir}/tpm"
+pseudobulk_dir="${data_dir}/pseudobulk"
```

Comment:
I don't think these are used yet here?

Reply:
No, they aren't use yet, but will be in next steps. I could delete these lines now but they'll just get added back in the next PR, so I think it's fine to keep here?

Reply:
I'm good with keeping them, just wanted to check!